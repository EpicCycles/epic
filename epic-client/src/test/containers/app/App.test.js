import React from 'react';
import createHistory from "history/createBrowserHistory";
import App from "../../../containers/app";

describe("App.index tests", () => {
    it('renders the App correctly', () => {
        const history = createHistory();
        history.push('/');
        const app = shallow(<App history={history}  />);
        expect(app).toMatchSnapshot();
    });
});
