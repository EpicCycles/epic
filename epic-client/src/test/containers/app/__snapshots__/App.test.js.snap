// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`App.index tests renders the App correctly 1`] = `
<div>
  <ul
    className="nav"
  >
    <li
      className="dropdown"
    >
      <Link
        className="dropbtn"
        replace={false}
        to="/"
      >
        Home
      </Link>
    </li>
    <li
      className="dropdown"
    >
      <Link
        className="dropbtn"
        replace={false}
        to="/about-us"
      >
        About
      </Link>
    </li>
    <li
      className="dropdown"
    >
      <p
        className="dropbtn"
      >
        Customer
      </p>
      <div
        className="dropdown-content"
      >
        <div
          className="dropdown-section"
        >
          Customer
        </div>
        <Link
          className="dropbtn"
          replace={false}
          to="/customer-search"
        >
          Find Customer
        </Link>
      </div>
    </li>
  </ul>
  <main>
    <Route
      component={[Function]}
      exact={true}
      path="/"
    />
    <Route
      component={[Function]}
      exact={true}
      path="/404"
    />
    <Route
      component={[Function]}
      exact={true}
      path="/about-us"
    />
    <Route
      component={[Function]}
      exact={true}
      path="/customer"
    />
    <Route
      component={[Function]}
      exact={true}
      path="/customer-search"
    />
  </main>
</div>
`;
